TUTORIAL.md

```diff
@@ -22,15 +22,19 @@ This tutorial will be easiest if you have some knowledge of how to use Linux com
 Once those three setup items are taken care of, you're ready to start learning how to use Spotipy!
 
 ## Step 1. Creating a Spotify Account
-Spotipy relies on the Spotify API. In order to use the Spotify API, you'll need to create a Spotify developer account.
+Spotipy relies on the Spotify API. In order to use the Spotify API, you'll 
+need to create a Spotify developer account. Here is a how-to [video](https://www.youtube.com/watch?v=3RGm4jALukM) that showcases how to get 
```

Comment:
this video might be useful but the image is very low quality and hard to read. I would rather not include this in official docs.